homeassistant/components/camera/familyhub.py

```diff
@@ -0,0 +1,60 @@
+"""
+Family Hub camera for Samsung Refrigerators.
+
+For more details about this platform, please refer to the documentation
+https://home-assistant.io/components/camera.familyhub/
+"""
+import logging
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (CONF_NAME)
+from homeassistant.components.camera import Camera
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+_LOGGER = logging.getLogger(__name__)
+
+REQUIREMENTS = ['pyfamilyhublocal==0.0.2']
+
+CONF_IP = 'address'
```

Comment:
We have `CONF_IP_ADDRESS` in `const.py`. Please use that.

Reply:
Got it, and i'm updating the docs now